bin/table_classification_summaries_main

```diff
@@ -111,7 +111,7 @@ def main(data_dcts, window_size, latex_file, with_preamble=False):
                     median_t, error_t = bootstrap_confidence_interval(time_to_steadys)
                     time_to_steady = format_median_error(median_t, error_t, brief=True)
                 else:  # No changepoints in any process executions.
-                    mean_last_cpt = format_median_error(median, 0, as_integer=True)
+                    mean_last_cpt = format_median_error(0, 0, as_integer=True)
```

Comment:
yes, it was showing the integer part of "steady state perf", if you look in the b3 table, this is 1.64 for PyPy / n-body (steady iter shows as 1) and 2.6 for Richards / LuaJIT (steady iter shows up as 2)